.github/workflows/dev-image-update.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
           token: ${{ secrets.BROADBOT_TOKEN }}
           path: datarepo-helm-definitions
       - name: "Bump the tag to a new version"
-        uses: broadinstitute/datarepo-actions/actions/main@0.57.0
+        uses: broadinstitute/datarepo-actions/actions/main@0.59.0
```

Comment:
Is it worth updating version if we know it has no effect? Better to keep them all the same, I guess?

Reply:
That's been my approach - I'd rather keep everything on the same version. We've run into issues before when some actions get behind in version and then we don't realize we're introducing errors until we actually need to make a change. It gets messy.